lock/src/main/java/com/auth0/android/lock/provider/OAuth2WebAuthProvider.java

```diff
@@ -105,18 +114,11 @@ public void setParameters(Map<String, Object> parameters) {
     @Override
     protected void requestAuth(Activity activity, String connectionName) {
         if (account.getAuthorizeUrl() == null) {
-            if (callback != null) {
-                callback.onFailure(R.string.com_auth0_lock_social_error_title, R.string.com_auth0_lock_social_invalid_authorize_url, null);
-            } else {
-                Log.w(TAG, "No callback set for web IdP authenticator");
-            }
+            callback.onFailure(R.string.com_auth0_lock_social_error_title, R.string.com_auth0_lock_social_invalid_authorize_url, null);
             return;
         }
 
-        //Generate random state
-        lastState = UUID.randomUUID().toString();
```

Comment:
I've moved it to https://github.com/auth0/Lock.Android/pull/243/files#diff-f28e3e975bb7b4fcd480dc5a7c55ecafR222